docs/running.rst

```diff
@@ -51,9 +51,5 @@ Init scripts
   <https://www.archlinux.org/packages/community/any/mopidy/>`__ comes with a systemd init
   script.
 
-- A blog post by Benjamin Guillet explains how to `Daemonize Mopidy and Launch
-  It at Login on OS X
-  <http://www.benjaminguillet.com/blog/2013/08/16/launch-mopidy-at-login-on-os-x/>`_.
-
```

Comment:
Nice catch. This has been broken for a long time, ref #887, and the Homebrew formula supports this out of the box these days.